addons/discuss-button/userscript.js

```diff
@@ -1,5 +1,6 @@
 export default async function ({ addon, global, console }) {
   if (!addon.tab.clientVersion) return;
+  if (document.querySelector(".mod-conference")) return;
```

Comment:
Please look at the URL instead, this element is rendered by React and might not be ready by the time this addon runs

Reply:
That'll do?

Reply:
Kinda feel like this isn't the optimal way however, it's fine for now until we find another navbar we shouldn't be appending to.